content/applications/general/iot/config/updating_iot.rst

```diff
@@ -73,6 +73,92 @@ Lastly, click :guilabel:`Flash`, and wait for the process to finish.
    An alternative software for flashing the micro SD card is `Raspberry Pi Imager
    <https://www.raspberrypi.com/software/>`_.
 
+Windows IoT update
+==================
+
+Windows virtual :abbr:`IoT (Internet of Things)` box may occasionally need an update to work
+properly.
+
+The following processes cover the uninstallation and re-installation of the Windows virtual
+:abbr:`IoT (Internet of Things)` box.
+
+.. important::
+   Before proceeding with the uninstallation of a previous version of the Windows virtual
+   :abbr:`IoT (Internet of Things)` box, check Odoo's latest download page to ensure a newer version
+   of Odoo's virtual :abbr:`IoT (Internet of Things)` box is available.
+
+   To reach the download site for the latest update, use the following URL:
+   `https://nightly.odoo.com/16.0/nightly/windows/`.
+
+   It is imperative that the `16.0` be replaced with the version of Odoo that the database is
+   running on (only full versions or long supported versions). The latest file has `latest` in its
+   filename.
+
+   Note the date of the upload to the right of the filename.
```

Comment:
@tiku-odoo I have mixed feelings regarding the changes. There is nothing wrong using the "Download" button on odoo.com/download.
For me the main issue is that the client will likely be confused as when you download the file it always say "..._latest" at the end. I can already see coming customers that will download the file a second time X months after and accidentally install the previously downloaded installed (so with older code) instead of the new one.

Maybe we could recommend to download any existing installer in the download folder before hand to avoid that

Reply:
Updated with warning admonition block to avoid this issue.